libs/image-editor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt

```diff
@@ -153,7 +153,7 @@ class CropFragment : Fragment(), UCropFragmentCallback {
         }
     }
 
-    private fun showCropError(errorMsg: String?, errorResId: Int) {
+    private fun showCropError(errorResId: Int) {
         // TODO: track exact error errorMsg
```

Comment:
I think we should remove the `TODO` if we are removing the `errorMsg` parameter.

```suggestion
```